lib/msf/core/db_manager/http/servlet/exploit_servlet.rb

```diff
@@ -29,4 +29,6 @@ def self.report_exploit
       exec_report_job(request, &job)
     }
   end
-end
\ No newline at end of file
+
```

Comment:
Any reason for the added whitespace here?

Reply:
Nope, just oversight. I'll go fix that!